codecs/webp_dec/example.html

```diff
@@ -0,0 +1,44 @@
+<!doctype html>
+<script src='webp_dec.js'></script>
+<script>
+  const Module = webp_dec();
```

Comment:
I'm not going to review this file in terms of project types, but isn't this not-a-constructor, so should start with a lower-case?

The reason I ask, is I saw @developit doing a similar thing in the options PR. I'm worried I've missed out on some change to consensus I always thought it was:

```
ThisIsAClass
THIS_IS_A_PRIMITIVE_CONSTANT
thisIsEverythingElse
```

Reply:
It’s mostly this way ’cause that’s how all the Emscripten code examples are written. By default, Emscripten glue creates a global called `Module`. We set a flag so that instead it registers a function that returns that `Module`, hence the naming. It’s nbd to change that, tho